cpp/turbodbc/Library/src/command.cpp

```diff
@@ -62,4 +62,9 @@ int64_t command::get_row_count()
     }
 }
 
+void command::finalize()
+{
+    const_cast<cpp_odbc::statement &>(*statement_).finalize();
```

Comment:
This is really mean ;-). Please drop the const-cast here and mark the `finalize` function as `const`. The `finalized` member then must be declared `mutable`, but I prefer that to the const-cast here.

Reply:
It's not as simple: statement::finalize is also not const, because it needs non-const access on its inner handle. So one could go "all the way const", but that would also be weird, because "finalize" certain is (semantically speaking) a non-const operation.